packages/react-native-executorch/common/rnexecutorch/models/vertical_ocr/VerticalOCR.cpp

```diff
@@ -0,0 +1,125 @@
+#include "VerticalOCR.h"
+#include <future>
+#include <rnexecutorch/data_processing/ImageProcessing.h>
+#include <rnexecutorch/data_processing/Numerical.h>
+#include <rnexecutorch/models/ocr/Types.h>
+#include <vector>
+
+namespace rnexecutorch {
+
+VerticalOCR::VerticalOCR(const std::string &detectorLargeSource,
+                         const std::string &detectorNarrowSource,
+                         const std::string &recognizerSource,
+                         const std::string symbols, const bool independentChars,
```

Comment:
```suggestion
                         std::string symbols, bool independentChars,
```

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html the same for header